app/views/transactions/index.html.haml

```diff
@@ -0,0 +1,21 @@
+-# TODO: app/views/accounts/transactions_in_review.html.haml is similar
+
+= render "shared/transactions/headers"
+
+- content_for :h1 do
+  = text("transactions.in_review")
+
+#header_billing.grid_9= render "shared/transactions/search"
+
+.clearfix
+
+- if @order_details.any?
+  .grid_12= render "shared/transactions/table", order_details: @order_details
+
+- if @recently_reviewed.any?
```

Comment:
Do we want to do anything if both `@order_details` and `@recently_reviewed` are both empty?